nzpyida/wrappers/decision_trees.py

```diff
@@ -33,8 +33,7 @@ def __init__(self, idadb: IdaDataBase, model_name: str):
     def fit(self, in_df: IdaDataFrame, id_column: str, target_column: str, in_column: str=None,
         col_def_type: str=None, col_def_role: str=None, col_properties_table: str=None,
         weights: str=None, eval_measure: str=None, min_improve: float=0.02, min_split: int=50,
-        max_depth: int=10, val_table: str=None, val_weights: str=None, qmeasure: str=None,
-        statistics: str=None):
+        max_depth: int=10, statistics: str=None):
```

Comment:
I was thinking about something slightly different - please leave this file alone, and I'll handle it.